sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -462,14 +452,16 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
 
   @AnyThread
   fun setMaximumFps(fps: Int) {
-    renderTimeNs = ONE_SECOND_NS / fps
+    renderHandlerThread.post {
+      fpsManager.updateMaxFps(fps)
```

Comment:
I'd name it somehow similar to `setScreenRefreshRate`, either `setScreenRefreshRate` / `setUserRefreshRate` or `setUserFps` / `setScreenFps`

Reply:
thanks, will align